docs/UserGuide.md

```diff
@@ -342,75 +342,97 @@ A success message will be displayed if the suppliers are successfully sorted.
+
+<box type="details" seamless>
 
 Parameters:
 
-- `on/DELIVERY_DATE_TIME`: Must be in dd-MM-yyyy hh:mm format, and must not be blank.
-- `s/SUPPLIER_INDEX`: Must be a number greater than 0, and must not be blank.
-- `pro/PRODUCT`: Must be alphanumeric, can include spaces but must not start with a space, and must be between 1 and 50 (inclusive) characters long.
-- `q/QUANTITY`: Must be a number greater than 0 followed by a space, and unit and must not be blank.
-- `c/COST`: Must be a number greater than 0 with up to 2 decimal places allowed, and must not be blank.
-<box type="tip" seamless>
+- `on/DELIVERY_DATE_TIME`: `DELIVERY_DATE_TIME` is the date and time of delivery. It must be in dd-MM-yyyy hh:mm format, and cannot be blank.
+- `s/SUPPLIER_INDEX`:`SUPPLIER_INDEX` is the index of supplier currently displayed. It must be a number between 1 and the total number of suppliers currently displayed (inclusive), and cannot be blank.
+- `pro/PRODUCT`: `PRODUCT` is the product associated with the delivery. It must be alphanumeric, only contain between 1 and 50 (inclusive) characters, and spaces are also allowed.
+    - One space is counted as one character.
+    - `PRODUCT` cannot be made up of only spaces.
+- `q/QUANTITY`: `QUANTITY` is the amount of product to be delivered with units. It must be a number greater than 0 followed by a space, and a unit and must not be blank.
+    - `QUANTITY` cannot have decimal places.
+    - `QUANTITY` units are case sensitive.
+- `c/COST`: `COST` is the total cost for the delivery. It must be a number greater than 0 with up to 2 decimal places allowed, and cannot be blank.
+
+**Tips:** 
+- Day and month of DELIVERY_DATE_TIME must be in double digits!
 
-**Tip:** Day and month of date must be in double digits!
 </box>
 
 <box type="warning" seamless>
 
 **Warnings**:
-- A spacing between `add` and `-d` is compulsory
-- A delivery is considered duplicate if they have the same DATE, TIME, SUPPLIER, PRODUCT and COST
-- Duplicate delivery will not be added again
-- No duplicate prefix can be used
-- Units for QUANTITY is case-sensitive
+- At least one space is needed between `add` and `-d`.
+- A warning will be given if the user tries to add a duplicate delivery.
+- A delivery is considered duplicate and will not be added again if it has the same `DELIVERY_DATE_TIME`, `SUPPLIER`, `PRODUCT`, `QUANTITY`, `COST` and `STATUS` as an existing delivery.
+    - Comparison between different `PRODUCT`is case-sensitive.
+- A delivery has a default `STATUS` of `PENDING`.
+
 </box>
 
-#### Example
+Examples:
+- `add -d on/18-06-2024 17:00 s/1 pro/bread q/500 g c/25.50`
+- `add -d on/19-12-2022 08:00 s/2 pro/rice q/50 kg c/50.20 `
 
-    add -d on/18-06-2024 17:00 s/1 pro/bread q/500 g c/5.50
+Expected output:
+- Delivery details is shown and paired to supplier at index 1, assuming there is at least one supplier. Otherwise, an error message will be shown.
```

Comment:
do you think it will be more intuitive and user friendly if we combine every example with its expected output instead of separating the example commands and the expected outputs?